doc/src/contribute/faq.md

```diff
@@ -93,7 +95,7 @@ We have a number of examples and demos available for viewing at: [https://docs.s
 
 ### Are you looking for partners?
 
-We are seeking partners that can contribute to the ecosystem primarily in development by building apps with the SDK now so they can be ready to launch when the network goes live.
+We are seeking partners that can contribute to the ecosystem primarily in development by building apps with the SDK now so they can be ready to launch when the network goes live. If interested, please apply here: https://bit.ly/suiform
```

Comment:
```suggestion
We are seeking partners that can contribute to the ecosystem primarily in development by building apps with the SDK now so they can be ready to launch when the network goes live. If interested, please apply [here]( https://bit.ly/suiform)
```

Reply:
Haha, I thought they wanted a short link to expose the tiny URL. :-)